Content.Server/Weapon/Melee/MeleeWeaponSystem.cs

```diff
@@ -107,7 +109,14 @@ private void OnClickAttack(EntityUid owner, MeleeWeaponComponent comp, ClickAtta
                                 $"{ToPrettyString(args.User):user} melee attacked {ToPrettyString(args.Target.Value):target} using {ToPrettyString(args.Used):used} and dealt {damageResult.Total:damage} damage");
                     }
 
-                    SoundSystem.Play(Filter.Pvs(owner), comp.HitSound.GetSound(), target);
+                    if (hitEvent.HitSoundOverride != null)
+                    {
+                        SoundSystem.Play(Filter.Pvs(owner), hitEvent.HitSoundOverride.GetSound(), target, AudioHelpers.WithVariation(0.25f));
+                    } else
+                    {
```

Comment:
```suggestion
                    } 
                    else
                    {
```

Reply:
Or maybe just abbreviate this by using something like
```cs
var sound = hitEvent.HitSoundOverride ?? comp.HitSound
if (sound != null)
	SoundSystem.Play(....)
```

Reply:
Would this abbreviation work even with one of the Plays() taking in a WithVariation? 
One is Play(owner, sound, target, withvariation) and the other is just  Play(owner, sound, target)